microservices/clients/contestproposal-client/src/main/java/org/xcolab/client/proposals/ProposalAttributeClient.java

```diff
@@ -52,10 +52,6 @@ public ProposalAttribute createProposalAttribute(ProposalAttribute proposalAttri
                 .execute().toPojo(serviceNamespace);
     }
 
-    public ProposalAttribute getImpactProposalAttributes(Long proposalId) {
-        return null;
-    }
-
```

Comment:
This change seems unrelated.

Reply:
You're right, this commit is on the wrong branch. You can ignore this, the change has no functional impact as the method was unused.